### ISSUE TITLE
Windows "Index out of range."

### ISSUE THREAD

**Issue by Unselfish** [haskell/ThreadScope#108]
I'm having trouble opening event logs with the threadscope 0.2.11 Windows binary. Using GHC 8.6.5.

![image](https://user-images.githubusercontent.com/40940869/77338872-58c63100-6d01-11ea-88d4-df15eacb288b.png)

I followed the instructions here to set up. https://github.com/haskell/ThreadScope/issues/52. My threadscope.exe is in the `/bin` folder of gtk2 with all the .dll files.

**Comment by maoe** [comment]
Do you get that error when opening an eventlog, or just starting threadscope without an eventlog? If the former is the case can you upload the eventlog file?

**Comment by Unselfish** [comment]
The error occurs when opening an eventlog. I've only tried my own ones. Testing with `--test ch8` works.

[The event log.](https://github.com/haskell/ThreadScope/files/4372530/program.txt)

(Changed it to a text file for uploading)

**Comment by maoe** [comment]
Thanks for sharing your eventlog. I could reproduce the issue with just `ghc-event inc`. So it's not a threadscope or Windows-specific issue.

I've file an issue at the ghc-events issue tracker: https://github.com/haskell/ghc-events/issues/58

@Unselfish Do you have any other eventlog files that cause this issue?

**Comment by Unselfish** [comment]
Yes. All the eventlogs I've made have this issue, with a different index per log.

[log2.txt](https://github.com/haskell/ThreadScope/files/4376409/log2.txt)
[log3.txt](https://github.com/haskell/ThreadScope/files/4376421/log3.txt)
[log4.txt](https://github.com/haskell/ThreadScope/files/4376429/log4.txt)
